rockspec/lua-tinyyaml-0.1-0.rockspec

```diff
@@ -1,15 +1,15 @@
-package = "lua-tinyyaml"
-version = "0.1-0"
+package = "api7-lua-tinyyaml"
```

Comment:
we can use this name `lua-tinyyaml`, if we need to release a new version, we can release a new version, eg: v2.0